enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer/003_kube_enforcer_deploy.yaml

```diff
@@ -93,65 +93,102 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: starboard-operator
+  name: trivy-operator
   namespace: aqua
   labels:
-    app: starboard-operator
+    app.kubernetes.io/name: trivy-operator
+    app.kubernetes.io/instance: trivy-operator
+    app.kubernetes.io/version: "0.25.0"
+    app.kubernetes.io/managed-by: kubectl
 spec:
   replicas: 1
   strategy:
     type: Recreate
   selector:
     matchLabels:
-      app: starboard-operator
+      app.kubernetes.io/name: trivy-operator
+      app.kubernetes.io/instance: trivy-operator
   template:
     metadata:
       labels:
-        app: starboard-operator
+        app.kubernetes.io/name: trivy-operator
+        app.kubernetes.io/instance: trivy-operator
     spec:
-      serviceAccountName: starboard-operator
+      serviceAccountName: trivy-operator
       automountServiceAccountToken: true
-      securityContext: {}
       containers:
-        - name: operator
-          image: docker.io/aquasec/starboard-operator:0.15.25
+        - name: "trivy-operator"
+          image: "docker.io/aquasec/trivy-operator:0.25.0"
           imagePullPolicy: IfNotPresent
-          securityContext:
-            privileged: false
-            readOnlyRootFilesystem: true
-            allowPrivilegeEscalation: false
-            capabilities:
-              drop:
-                - ALL
           env:
             - name: OPERATOR_NAMESPACE
               value: aqua
             - name: OPERATOR_TARGET_NAMESPACES
               value: ""
+            - name: OPERATOR_EXCLUDE_NAMESPACES
+              value: "kube-system"
+            - name: OPERATOR_TARGET_WORKLOADS
+              value: "pod,replicaset,replicationcontroller,statefulset,daemonset,cronjob,job"
+            - name: OPERATOR_SERVICE_ACCOUNT
+              value: "trivy-operator"
             - name: OPERATOR_LOG_DEV_MODE
-              value: "false"
+              value: "true"
```

Comment:
it should be false as default

Reply:
@mjshastha , Should it be false or true?